src/FormValidation.js

```diff
@@ -0,0 +1,32 @@
+import * as yup from "yup";
+
+export const signInValidationSchema = yup.object({
+    email: yup
+      .string("Enter your email")
+      .email("Enter a valid email")
+      .required("required"),
+    password: yup
+      .string("Enter your password")
+      .min(8, "Password should be of minimum 8 characters length")
+      .required("required"),
+  });
```

Comment:
First, good job on separating out these validation schemas. 💯 

I don't think it's necessary to set a placeholder/default expectant value in each key/value pair. Here's a snippet from the Formik docs that I think does the exact same thing with less code: 
```suggestion
  validationSchema={Yup.object({
         firstName: Yup.string()
           .max(15, 'Must be 15 characters or less')
           .required('Required'),
         lastName: Yup.string()
           .max(20, 'Must be 20 characters or less')
           .required('Required'),
         email: Yup.string().email('Invalid email address').required('Required'),
       })}
```

If you are wanting the default value to have `Enter your <fieldName>` that is fine and you can most likely ignore this comment, but I think it would be better to add that text as a placeholder attribute on the Field component itself.

Reply:
Done, good call out.

Reply:
Moved these validators back into each component per Liam's instruction.